fec.config.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-undef */
+const { resolve } = require('path');
+
+module.exports = {
+    appUrl: '/insights/vulnerability',
+    debug: true,
+    useProxy: process.env.PROXY === 'true',
+    proxyVerbose: true,
+    interceptChromeConfig: false,
+    plugins: [],
+    ...(process.env.HOT ? { hotReload: process.env.HOT === 'true' } : { hotReload: true }),
+    ...(process.env.port ? { port: parseInt(process.env.port) } : {}),
+    moduleFederation: {
+        shared: [
+            {
+                'react-router-dom': {
+                    singleton: true,
+                    import: false,
+                    version: '^6.8.1',
```

Comment:
Same as in the Patch PR: https://github.com/RedHatInsights/patchman-ui/pull/1178#discussion_r1559258671

Reply:
@bastilian I have pushed the necessary changes. PTAL